src/QueueReceiver.Infrastructure/Data/ApplicationDbContext.cs

```diff
@@ -1,10 +1,12 @@
 ﻿using Microsoft.EntityFrameworkCore;
+using QueueReceiver.Core.Interfaces;
 using QueueReceiver.Core.Models;
+using System.Threading.Tasks;
 using Person = QueueReceiver.Core.Models.Person;
```

Comment:
This is never a good sign... Why do we need to explicitly set using Person?

Reply:
Pretty sure this is my bad. It is not needed and may be removed.